Framework/DataHandling/test/LoadSampleShapeTest.h

```diff
@@ -167,6 +243,14 @@ class LoadSampleShapeTest : public CxxTest::TestSuite {
     TSM_ASSERT_DIFFERS("Shape is not a mesh object", mObj, nullptr);
     return mObj;
   }
+
+  std::unique_ptr<MeshObject> loadCube() {
```

Comment:
The rotation tests should just use a manually constructed `MeshObject`.

This narrows down the functionality tested to what you actually want to test (rotation of a mesh by the goniometer of a workspace) and means you don't have to inspect the STL file to read the test data.

As a general rule, if you can avoid having to load files in unit tests, do.